pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>com.facebook.airlift</groupId>
         <artifactId>airbase</artifactId>
-        <version>99</version>
+        <version>100</version>
```

Comment:
IIRC this involved a jackson module upgrade as well. Does compilation goes through successfully after bumping this version.?

Reply:
I think there were 2 places that need update because of this and is included in the PR.